test/app-tap/tarantoolctl.test.lua

```diff
@@ -138,7 +134,8 @@ local function tctl_command(dir, cmd, args)
     if not fio.stat(fio.pathjoin(dir, '.tarantoolctl')) then
         create_script(dir, '.tarantoolctl', tctlcfg_code)
     end
-    local command = ('LUA_PATH="' .. rocks_path .. '" tarantoolctl %s %s'):format(cmd, args)
+    local command = ('LUA_PATH="' .. rocks_path ..
```

Comment:
It will be more readable if it would be
```lua
local command = ('LUA_PATH="' .. rocks_path .. '" ' ..
    TARANTOOLCTL_PATH .. ' %s %s'):format(cmd, args)
```

Reply:
Fixed